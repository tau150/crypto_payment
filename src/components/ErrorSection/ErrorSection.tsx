import { MdError } from "react-icons/md";

import Card from "@/components/UI/Card/Card";

const ErrorSection = () => {
  return (
    <Card className="h-60 w-full">
      <h1 className="flex mb-10 justify-center">
        <MdError className="text-red-500 text-2xl mr-3" />
        Ooops!
      </h1>
      <h1>Something went wrong, please try again later.</h1>
    </Card>
  );
};

export default ErrorSection;
