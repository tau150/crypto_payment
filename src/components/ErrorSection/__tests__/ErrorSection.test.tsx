import React from "react";
import { render } from "@testing-library/react";

import ErrorSection from "../ErrorSection";

test("renders ErrorSection correctly", () => {
  const { container } = render(<ErrorSection />);

  expect(container.firstChild).toMatchSnapshot();
});
