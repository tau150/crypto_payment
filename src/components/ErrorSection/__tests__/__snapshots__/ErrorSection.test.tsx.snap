// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorSection correctly 1`] = `
<div
  class="shadow-md p-6 border-slate-100 rounded-lg border  h-60 w-full"
>
  <h1
    class="flex mb-10 justify-center"
  >
    <svg
      class="text-red-500 text-2xl mr-3"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
      />
    </svg>
    Ooops!
  </h1>
  <h1>
    Something went wrong, please try again later.
  </h1>
</div>
`;
